### ISSUE TITLE
IMDb links should open in a new tab

### ISSUE THREAD

**Issue by AndreasPB** [streamchaser/streamchaser#255]
### Is your feature request related to a problem? Please describe.

People might just leave the page if their tab changes

### Describe the solution you'd like

The IMDb link should open a new tab

**Comment by Primdahl26** [comment]
My counter argument would be that if you want to open in a new tab, you could always use ctrl click.

But you cant open in current tab if we make it default to a new tab i Think.

**Comment by AndreasPB** [comment]
> My counter argument would be that if you want to open in a new tab, you could always use ctrl click.
> 
> But you cant open in current tab if we make it default to a new tab i Think.

True, but I think this is one of the cases where we might want to limit what can happen. People getting a different site in "our" tab has a high chance of them just leaving

I'm not hard for our against, so we can discuss this a bit
